client/src/components/Pages/Match.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+    <layout>
+        <h1 class="text-3xl block mb-4">Match aanmaken</h1>
+        <label
+            class="block text-gray-700 text-sm font-bold mb-2"
+            for="checkbox"
+        >
+            Type match
+        </label>
+        <div class="inline-block relative w-full mr-8">
+            <select
+                class="block appearance-none w-full bg-white border border-gray-400 hover:border-gray-500 px-4 py-2 pr-8 rounded shadow leading-tight focus:outline-none focus:shadow-outline"
```

Comment:
This is how tailwind works. Instead of creating new classes to implement these (which goes against the way tailwind works) it will be better to seperate it in a custom select box vue file. However I haven't found a good way to implement that dynamically. 

It might be good to create a class while I'm figuring out how to implement a custom select box though..

Reply:
It can be quick and easy, but I am not really a fan of it, because it reflects bad practices.